pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -189,28 +189,15 @@ func (s *apiScaffolder) scaffoldCreateAPIFromPlugins(isLegacyLayout bool) error
 // which will have its own controller template which set the recorder so that we can use it
 // in the reconciliation to create an event inside for the finalizer
 func (s *apiScaffolder) updateMainByAddingEventRecorder(isGoV3 bool, defaultMainPath string) error {
```

Comment:
Wondering if we can drop the parameter `isGoV3 bool` if it is no longer used?

Reply:
Yep good point.
I changed that